src/server/session.ts

```diff
@@ -1895,14 +1900,27 @@ namespace ts.server {
 
         private getFullNavigateToItems(args: protocol.NavtoRequestArgs): readonly NavigateToItem[] {
             const { currentFileOnly, searchValue, maxResultCount } = args;
+            if (!args.file) {
+                const items: NavigateToItem[] = [];
+                this.projectService.forEachEnabledProject(project => {

```

Comment:
Why not use `combineProjectOutputFromEveryProject` You shouldne be needing to load ancestor project tree during every project callback

Reply:
I didn't know about that, thanks!